examples/gno.land/p/demo/ufmt/ufmt_test.gno

```diff
@@ -38,6 +38,7 @@ func TestSprintf(t *testing.T) {
 		{"no args", nil, "no args"},
 		{"finish with %", nil, "finish with %"},
 		{"stringer [%s]", []interface{}{stringer{}}, "stringer [I'm a stringer]"},
+		{"Hello, World! 😊", nil, "Hello, World! 😊"},
```

Comment:
It's not critical, but I think it would be nice to add the characters shown as an example in #1814 to the test.

```suggestion
		{"Hello, World! 😊", nil, "Hello, World! 😊"},
		{"â", nil, "â"},
```

Reply:
@notJoon thanks for the advice, I added the tests you suggested.